devel/openssl3/Portfile

```diff
@@ -14,7 +14,7 @@ set major_v         3
 epoch               1
 github.setup        openssl openssl ${major_v}.3.2 openssl-
 name                openssl3
-revision            0
+revision            1
```

Comment:
Since this is a build-only fix and OpenSSL either built correctly, or didn't build at all, I don't think the bump of the revision is necessary.
Since the installed files do not change, I also don't think we need to bump the dependents.